addons/mail/static/src/core/common/thread.xml

```diff
@@ -39,6 +39,10 @@
                         onParentMessageClick="() => msg.parentMessage and env.messageHighlight?.highlightMessage(msg.parentMessage, props.thread)"
                         thread="props.thread"
                         messageEdition="props.messageEdition"
+                        avatarPlaceholder="props.avatarPlaceholder"
+                        closeThread="!props.threadResolved and msg_first"
+                        openThread="props.threadResolved and msg_first"
```

Comment:
`openThread` and `closeThread` are passed with `env` in knowledge so they must be removed